packages/react-internal/src/components/Persona/Persona.types.ts

```diff
@@ -86,6 +86,10 @@ export interface IPersonaSharedProps extends React.HTMLAttributes<HTMLDivElement
    * @defaultvalue Derived from `text`
    */
   initialsColor?: PersonaInitialsColor | string;
+  /**
```

Comment:
nit: lets add a newline before the comment to keep it consistent with the other props